vibe_code_apps_22/eplus_native/schedule_calendar_repair.py

```
@@ -325,6 +325,302 @@ def _patch_htg_setpoints(text: str, contract: dict[str, Any]) -> str:
+    fields: list[str] = ["Through: 12/31"]
+    for key in day_order:
+        day = days.get(key) or {}
+        fields.append(f"For: {for_names[key]}")
+        if not day.get("occupied", False):
+            fields.extend(["Until: 24:00", un])
+            continue
+        start = str(day.get(start_key) or "06:00")
+        end = str(day.get(end_key) or "18:00")
+        if lead_hours > 0 and start_key.startswith("hvac"):
+            people_start = str(day.get(people_start_key) or start)
+            pulled = _shift_hhmm_str(people_start, -int(round(float(lead_hours) * 60)))
```

Ruff (0.16.1)

[warning] Value being cast to `int` is already an integer

Remove unnecessary `int` call

(RUF046)